src/physics.F90

```diff
@@ -53,7 +53,7 @@ subroutine collision(p)
     end if
 
     ! check for very low energy
-    if (p % E < 1.0e-100_8) then
+    if (p % E < MIN_ENERGY) then
```

Comment:
I pulled your fork and tried to build and it broke on this line. I just submitted a PR to your fork which corrects this as well as updates your ``prodndpp`` branch with the latest version of the ``develop`` branch.